publishable/config/voyager.php

```diff
@@ -1,6 +1,46 @@
 <?php
 
 return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Factory config
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify what classes should be returned when using e.g Voyager::model('Category')
+    |
+    */
+
+    'factory' => [
```

Comment:
Where did the "factory" name come from? I'm not opposed to it, just curious

Reply:
I had a hard time naming this config because of a conflict in the config-file, where the key is used to change the default namespace of BREAD. So I had to think twice about what it actually was, and factory came to mind.